### PR TITLE
Create components 

### PR DESCRIPTION
**Pull request by tarek102**
### In this PR:

- Create Calculator.js
- Design Calculator

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,9 @@
...
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Calculator />
```
Comment (elmar8287) [inline src/App.js]: - [ ] Please, consider that the App.js should be a class-based component, as it is required in the project.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+function Calculator() {
+  return (
```
Comment (elmar8287) [inline src/components/Calculator.js]: - [ ]  Please, consider that the Calculator.js should be a class-based component, as it is required in the project.

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, @tarek102.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Nice PR description 👍 

## Required Changes ♻️

- [ ] Check the comments under the review.

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
### Please, remember to tag me (@elmar8287) in your question so I can receive the notification 👍 
giphy

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by addod19 — APPROVED** [review]
### Hi @tarek102, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Highlights
- [ ] Great job refactoring the function components into a class based components :100: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**